src/routers/management/llamacloud.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import APIRouter, UploadFile, Depends
+from fastapi.responses import JSONResponse
+from src.models.file import File
+from src.controllers.files import FileHandler, UnsupportedFileExtensionError
+from src.models.llamacloud_config import LlamaCloudConfig
+from src.controllers.env_configs import EnvConfigManager
+
+llamacloud_router = r = APIRouter()
+
+
+@r.get("")
+def get_llamacloud_config(
+    config: LlamaCloudConfig = Depends(LlamaCloudConfig.get_config),
+):
+    # TODO: call llamacloud API to get dashboard url
+    return config.to_api_response()
+
+
+@r.put("")
+def update_llamacloud_config(
+    new_config: LlamaCloudConfig,
+    config: LlamaCloudConfig = Depends(LlamaCloudConfig.get_config),
+):
+    print(new_config)
+    EnvConfigManager.update(config, new_config, rollback_on_failure=True)
+    return JSONResponse(
+        {
+            "message": "Config updated successfully.",
+            "data": new_config.to_api_response(),
+        }
+    )
```

Comment:
Once LlamaCloud is set up and enabled, the users no longer use the local index or upload files. I think we should have an update for that then.

Reply:
And don't forget, we have the code to switch vector store: https://github.com/ragapp/ragapp/blob/84bd368cc33f7667e47afafba73e412c95d60fd1/patch/backend/app/engine/vectordb.py#L8
You may need to update the VECTOR_STORE_PROVIDER too.